tensorrt_llm/_torch/models/modeling_nemotron_h.py

```diff
@@ -3,20 +3,16 @@
 import torch
 from torch import nn
 from torch.nn import functional as F
-
-try:
-    from transformer_engine.pytorch import RMSNorm
-except ImportError:
-    RMSNorm = None
 from transformers import AutoConfig, PretrainedConfig
 
 from ..attention_backend import AttentionMetadata
 from ..model_config import ModelConfig
 from ..modules.attention import Attention
 from ..modules.decoder_layer import DecoderLayer
 from ..modules.embedding import Embedding
-from ..modules.mamba.mixer import MambaMixer
+from ..modules.mamba.mixer import Mamba2Mixer
```

Comment:
This failed to import for me, I changed to `from ..modules.mamba.mamba2_mixer import Mamba2Mixer`and it worked.

Reply:
Good catch thanks! fixed